### ISSUE TITLE
Add wine menu entry to the start menu

### ISSUE THREAD

**Issue by V10lator** [lxqt/lxqt#413]
I really hate it that all my wine applications are listed at Sonstige (should be something like "others" in english). With XFCE there's a Wine submenu and even pcmanfm-qt shows it ( menu://applications/Wine ). So please sync pcmanfm-qt and the start menu by adding that wine submenu.

**Comment by dglent** [comment]
I have some applications that they are not listed in the menu, eg virtualbox, kmix, qt-recordmydesktop ....
in menu://applications/ in pcmanfm-qt i have all applications
it uses the /etc/xdg/menus/lxde-applications.menu

**Comment by psyca** [comment]
Still have this in Wine
"Empty entry" > "Empty entry" > Wine > ....

**Comment by szg0000** [comment]
After installing wine, and rebooting PC I have Wine menuitem.
Question: Why LXQT needs to be rebooted to see new menuitem?

**Comment by pmattern** [comment]
> Question: Why LXQT needs to be rebooted to see new menuitem?

Probably rebooting would not have been necessary but you were facing the faulty updating of the main menu within running LXQt sessions, see #65 which hints at a [workaround](https://github.com/lxde/lxqt/issues/65#issuecomment-137960481) as well.

The gents still suffering from the actual problem discussed in this issue may want to post some relevant information like the desktop entry files `*.directory` and `*.desktop` which are involved.

**Comment by stefonarch** [comment]
I spent an insane amount of fiddling with that including installing wine and irfanview until I noticed that the "wine" category is shown perfectly fine in both mainmenu  and fancymenu (without submenus here) with  `lxqt-applications.menu`.  With all others wine entries are shown under "other".

**Comment by tsujan** [comment]
> until I noticed that the "wine" category is shown perfectly fine

Yes, of course. I've had it here from time immemorial (although I almost never use it).
